shuup/admin/templates/shuup/admin/base.jinja

```diff
@@ -51,7 +52,13 @@
         {% block post_content %}{% endblock %}
         <script src="{{ static("shuup_admin/js/vendor.js") }}"></script>
         <script src="{{ static("shuup_admin/js/base.js") }}"></script>
+        <script src="{{ static("shuup_admin/js/imagelightbox.js") }}"></script>
         {% block extra_js %}{% endblock %}
+        <script>
```

Comment:
I know this script is tiny but we should try to keep base files clean. This should go into its own js file - see base/js/tooltip.js or base/js/popover.js for similar tiny scripts :)